source/nutrition/food_database.cpp

```diff
@@ -81,4 +82,9 @@ namespace nutrition {
     const std::vector<nutrition_facts> &food_database::foods() const { return foods_; }
     void food_database::foods(const std::vector<nutrition_facts> &foods) { foods_ = foods; }
     size_t food_database::size() const { return foods_.size(); }
+    void food_database::remove(unsigned food_index){foods_.erase(foods_.begin() + food_index);}
+    void food_database::update(unsigned food_index, nutrition_facts new_food){foods_[food_index] = std::move(new_food);}
+    void food_database::insert(const nutrition_facts& new_food) {foods_.push_back(new_food);}
+    nutrition_facts food_database::at(unsigned index){return foods_[index];}
```

Comment:
Use nomes padrão dos containers do C++.

`at` quando for checar se o index é válido e `operator[]` quando não for checar.